app/services/course/material/zip_download_service.rb

```diff
@@ -1,12 +1,68 @@
 class Course::Material::ZipDownloadService
+  require 'zip'
```

Comment:
@lowjoel You said I should require this at the gemfile ?  What's the benefits of that ?